### ISSUE TITLE
Plot throws OverflowError when plotting all-zero time series

### ISSUE THREAD

**Issue by aceffer** [highfestiva/finplot#172]
Plot throws OverflowError when plotting all-zero time series:

> Time
2021-03-20 17:00:00    0.0
2021-03-20 18:00:00    0.0
2021-03-20 19:00:00    0.0
2021-03-20 20:00:00    0.0
2021-03-20 21:00:00    0.0
                      ... 
2021-05-01 05:00:00    0.0
2021-05-01 06:00:00    0.0
2021-05-01 07:00:00    0.0
2021-05-01 08:00:00    0.0
2021-05-01 09:00:00    0.0
Name: Close, Length: 1000, dtype: float64

Code snippet:
```
df['zeros'] = 0
fplt.plot(df['zeros'])
```

Stack trace:

```
finplot\__init__.py:2507: RuntimeWarning: divide by zero encountered in log10
  sd = min(3, sd+int(abs(np.log10(rngmax))))
Traceback (most recent call last):
  File "main.py", line 513, in plotBasis
    fplt.show(qt_exec=False)
  File "finplot\__init__.py", line 1707, in show
    refresh()
  File "finplot\__init__.py", line 1703, in refresh
    _mouse_moved(win, None)
  File "finplot\__init__.py", line 2326, in _mouse_moved
    ax.crosshair.update(point)
  File "finplot\__init__.py", line 456, in update
    ytext = _round_to_significant(rng, rngmax, y, sd, se)
  File "finplot\__init__.py", line 2507, in _round_to_significant
    sd = min(3, sd+int(abs(np.log10(rngmax))))
OverflowError: cannot convert float infinity to integer
python-BaseException
```

**Comment by highfestiva** [comment]
The fix is easy, please supply a pull request bug fix instead of just huting for bugs.

**Comment by aceffer** [comment]
Well yes, the fix of this particular bug seems easy. Changing line:

`sd = min(3, sd+int(abs(np.log10(rngmax))))`

to 

```
if rngmax > 0:
    sd = min(3, sd+int(abs(np.log10(rngmax))))
else:
    sd = min(3, sd)
```

in `_round_to_significant(rng, rngmax, x, significant_decimals, significant_eps)`

would probably solve it, but there might be other factors that I don't see now.

**Comment by highfestiva** [comment]
Considering that
```python
>>> np.log10(0.00000001)
-8.0
```
You probably want a try-except with `sd = -8` or so in the catch clause.

**Comment by highfestiva** [comment]
Fixed in ccecee7.
